pageserver/src/tenant/storage_layer/layer_desc.rs

```diff
@@ -106,4 +129,46 @@ impl PersistentLayerDesc {
             self.image_file_name().into()
         }
     }
+
+    pub fn get_lsn_range(&self) -> Range<Lsn> {
+        self.lsn_range.clone()
+    }
+
+    pub fn get_key_range(&self) -> Range<Key> {
+        self.key_range.clone()
+    }
+
+    pub fn get_timeline_id(&self) -> TimelineId {
+        self.timeline_id
+    }
+
+    pub fn get_tenant_id(&self) -> TenantId {
+        self.tenant_id
+    }
+
+    pub fn is_incremental(&self) -> bool {
+        self.is_incremental
+    }
+
+    pub fn is_delta(&self) -> bool {
+        self.is_delta
+    }
```

Comment:
This is to make the API compatible with the original layer API, so as to avoid blasting `timeline.rs` too much, avoiding potential merge conflicts. We can also remove this in the future. Added TODO.